### PR TITLE
Remove Sinatra dependency

### PR DESCRIPTION
**Pull request by sunny**
In the same spirit as https://github.com/mperham/sidekiq/pull/3069, this PR removes the Sinatra dependency so as to make this gem slimmer.

- [x] <del>Also, allows choosing the web server as suggested in #34</del>
- [x] Also, allows choosing the path as suggested in #30

### CODE REVIEW

**Comment by arturictus** [comment]
Hi @sunny, thanks for your PR, it's great.
I tested it but when I send `^C` it does not stop sidekiq, that means the server is traping the signals.
You can test it with this project: https://github.com/arturictus/sidekiq_alive_example
I added the server config in the version 2.0.2 #37.
This was working before with this code: https://github.com/arturictus/sidekiq_alive/blob/master/lib/sidekiq_alive.rb#L20-L34

**Comment by sunny** [comment]
Good catch! I've fixed that and it now responds to `^C^ correctly on `sidekiq_alive_example`.

**Comment by sunny** [comment]
I also added a configurable `path` option (see also #30).

File: README.md [hunk 1]
```diff
@@ -197,11 +197,19 @@ curl localhost:7433
...
   # default: 7433
   #
   #   config.port = 7433
 
+  # ==> Server path
+  # HTTP path to respond to.
+  # Can also be set with the environment variable SIDEKIQ_ALIVE_PATH.
+  # default: '/'
+  #
+  #   config.port = '/'
```
Comment (arturictus) [inline README.md:211]:
```suggestion
  #   config.path = '/'
```

**Comment by arturictus** [comment]
Looks great! :clap: 
If you just change the config we can merge it

**Comment by arturictus** [comment]
New version released with this changes `2.0.3` :rocket:

**Comment by sunny** [comment]
Yay, thank you @arturictus for maintaining sidekiq_alive! 🚀

**Pull request merged**